shared/js/views.jsx

```diff
@@ -796,6 +797,18 @@ loop.shared.views = (function(_, mozL10n) {
         return <div className="no-video"/>;
       }
 
+      if (this.props.shareCursor && this.props.screenSharingPaused) {
+        return (
+          <div className="room-inner-info-area">
```

Comment:
I'm not really happy with this here. Although we're not displaying the stream, I don't think we should be randomly tearing down the video element just as sharing is paused. Additionally, it feels more standalone specific at the moment, as it'll only be displayed on the standalone?

Could we instead move this to be handled in standalone in the StandaloneRoomInfoArea ? It might still need a display:none on the video element.

If its very difficult or doesn't make sense, I'll be happy to reconsider, but I'd like to see an attempted prototype or explanations first.